buildspec.yml

```diff
@@ -0,0 +1,75 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_VERSION: "1.3.0"
+    ACCOUNT: "142577830533"
+    AWS_DEFAULT_REGION: "us-west-2"
```

Comment:
blocks sandbox build tests from working in another region, and overwrites value set by codebuild

Reply:
Updated